lib/index.js

```diff
@@ -43,6 +44,11 @@ const init = (settings = {}) => {
 
   flow.use(json({ extended: true }));
 
+  flow.use((req, res, next) => {
+    req._id = uuid();
+    next();
+  });
+
   flow.use((req, res, next) => {
     transaction.start(db)
       .then(trx => {
```

Comment:
```suggestion
      .then(trx => {
        trx._id = uuid();
```

Reply:
Note, this would need a line adding [here](https://github.com/UKHomeOffice/taskflow/blob/master/lib/activitylog/hooks/logger.js#L7) something like:

```
model.req = model.transaction._id;
```

but I can't suggest that because it's not in the diff.

Reply:
good idea